src/github/githubRepository.ts

```diff
@@ -16,7 +16,6 @@ import { PRDocumentCommentProvider } from '../view/prDocumentCommentProvider';
 import { convertRESTPullRequestToRawPullRequest, parseGraphQLPullRequest } from './utils';
 import { PullRequestResponse, MentionableUsersResponse } from './graphql';
 const queries = require('./queries.gql');
-import axois, { AxiosResponse } from 'axios';

```

Comment:
It seems we aren't using this library anywhere else, would be good to remove it from the package.json and our ThirdPartyNotices